lib/util/include/hse_util/bloom_filter.h

```diff
@@ -1,16 +1,17 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 /*
- * Copyright (C) 2015-2020 Micron Technology, Inc.  All rights reserved.
+ * Copyright (C) 2015-2022 Micron Technology, Inc.  All rights reserved.
  */
 
 #ifndef HSE_PLATFORM_BLOOM_FILTER_H
 #define HSE_PLATFORM_BLOOM_FILTER_H
 
+#include <bsd/bitstring.h>
```

Comment:
Maybe better to use setbit, clrbit, ... stuff from /usr/include/sys/param.h which is native to Linux (but looks like they took it from BSD verbatim).

Reply:
This is used for OMF.  Whatever we use must gauranteed not to change over time.

Reply:
The bitstring stuff looks like a more formalized interface designed to alloc dynamic bitstrings and such, dunno if that's good or bad relative to the bit macros...